docs/pages/zkvm/sdk-benchmarking.mdx

```diff
@@ -0,0 +1,210 @@
+---
+title: "Nexus zkVM Benchmark"
+lang: en-US
+description: "Benchmark your Nexus project using the SDK"
+image: "/nexus-head.png"
+---
+
+import Image from "next/image";
+
+## Benchmark Host and Guest program
+
+
+This benchmark demonstrates the performance of Nexus zkVM using a Fibonacci sequence calculation as an example. We'll analyze both the guest program's RISC-V cycle count and the host program's execution time for various Fibonacci sequence lengths.
+
+
+The host program `src/main.rs` is:
+
+```rust
+use nexus_sdk::{
+    compile::CompileOpts,
+    nova::seq::{Generate, Nova, PP},
+    Local, Prover, Verifiable,
+};
+
+const PACKAGE: &str = "guest";
+
+fn generate_pp_and_compile() -> (PP, Nova<Local>) {
+    let pp: PP = PP::generate().expect("failed to generate parameters");
+
+    let mut opts = CompileOpts::new(PACKAGE);
+    opts.set_memlimit(8); // use an 8mb memory
```

Comment:
If there's no reason for the extra memory over the 4mb base then I'd remove this and rerun the benchmarks (they're unnecessarily slow).

Reply:
updated in 915ef4c